src/amplify-migration/amplifyUtils.ts

```diff
@@ -111,3 +150,21 @@ export function normaliseUrlsForAmplify(filePath: string) {
   }
   return filePath.toLocaleLowerCase();
 }
+
+export async function createStagingLiteBranch(
+  repoName: string,
+  buildSpec: string
+) {
+  const appId = await createAmplifyApp(repoName, buildSpec, true);
+}
+
+export async function protectBranch(repoPath: string) {
+  const command = new UpdateAppCommand({
+    enableBasicAuth: true,
+    appId: repoPath,
+    basicAuthCredentials: Buffer.from(
+      `user:${process.env.AMPLIFY_DEFAULT_PASSWORD}`
```

Comment:
this is going to be the same for all?

Reply:
Yes, the point here is *not* to implement privatised quickie, but set up the infra work when privatised quickie comes to light. We can programatically change the password thr API later when developing quickie 2.0

Reply:
got it, but these private sites' will deploy direct from staging(not lite) for now till quickie 2.0 right?

Reply:
yup!